### DISCUSSION TITLE
Connected to rendezvous point, discovering nodes in 'xmr-btc-swap-mainnet' namespace . 0.10.1

### DISCUSSION THREAD

**Discussion by danilka2712** [comit-network/xmr-btc-swap#896] [Q&A] [answered]
Then nothing happens, what could be the matter?

**Comment by binarybaron** [comment]
Which version is your cli and which rendezvous have you connected to?

**Comment by danilka2712** [comment]
10.1 "/dnsaddr/rendezvous.coblox.tech/p2p/12D3KooWQUt9DkNZxEn2R5ymJzWj15MpG6mTW84kyd8vDaRZi46o

**Comment by delta1** [comment, marked as answer]
Please try the latest version and use a current rendezvous, like [unstoppable swap](https://github.com/comit-network/xmr-btc-swap/blob/master/docs/asb/README.md#asb-discovery) or [xmrswapme](https://xmrswap.me/#rendezvous).
